modules/govuk_containers/manifests/terraboard.pp

```diff
@@ -0,0 +1,127 @@
+# == Class: govuk_containers::terraboard
+#
+# Install and run Terraboard in a Docker container with PostgreSQL backend
```

Comment:
Is it worth adding a note here about why we're using a dockerised postgres (it's just storing transient state which can be recovered from s3)?  Otherwise someone will see this in the future and wonder why this is different to everything else.

Reply:
Yup, done!